test/StakeManager.js

```diff
@@ -1416,5 +1416,35 @@ describe('StakeManager', function () {
       staker = await stakeManager.getStaker(4);
       assertBNEqual(prevStake, staker.stake, 'Inactivity penalties have been levied');
     });
+    it('should not levy inactivity penalities if stake less than minStake ', async function () {
+      await mineToNextEpoch();
+      let epoch = await getEpoch();
+      const stake = tokenAmount('10000');
+      await razor.transfer(signers[16].address, stake);
+      await razor.connect(signers[16]).approve(stakeManager.address, stake);
+      await stakeManager.connect(signers[16]).stake(epoch, stake);
+      const stakerId = await stakeManager.stakerIds(signers[16].address);
+      await mineToNextEpoch();
+
+      epoch = await getEpoch();
+      await stakeManager.setStakerStake(epoch, stakerId, 1, tokenAmount('500'));
+
+      const epochsJumped = GRACE_PERIOD + 4;
+      for (let i = 0; i < epochsJumped; i++) {
+        await mineToNextEpoch();
+      }
+
```

Comment:
hey, once grace period is over, inactivity penalties are applied only when you commit or unstake
this test would have hold true without this pr as well.
right ?
I think we should do unstable here

Reply:
I have unstaked at L1445

Reply:
cool